testing/transform_scan.cu

```diff
@@ -347,3 +347,57 @@ struct TestTransformScanToDiscardIterator
 };
 VariableUnitTest<TestTransformScanToDiscardIterator, IntegralTypes> TestTransformScanToDiscardIteratorInstance;
 
+template <typename T>
+struct TestRegressionGH1332
```

Comment:
I'd sort of prefer a descriptive name instead of a reference to the issue.

Reply:
Changed to `TestValueCategoryDeduction`.